package.json

```diff
@@ -1,7 +1,8 @@
 {
   "private": true,
+  "type": "module",
```

Comment:
let’s please use .mjs for esm instead of type module

Reply:
done